src/jobs/jobs.controller.js

```diff
@@ -38,17 +40,31 @@ async function getAll() {
 /**
  * Transform an job from the database to a public object availiable in the API.
  */
-function _createPublicJob(job) {
+function _createPublicJob(job, teamId) {
```

Comment:
I disagree, offers should be unique, no matter the team which created them, same for the votes